tests/neighbor_advertiser_test.py

```diff
@@ -1,6 +1,7 @@
 import sys
 import os
 import pytest
+import mock
```

Comment:
In Python 3, `mock` is now part of the `unittest` module in the Python standard library. Please use `from unittest import mock` instead.